core/src/autogluon/core/metrics/__init__.py

```diff
@@ -413,6 +413,15 @@ def make_scorer(
 mean_absolute_percentage_error.add_alias("mape")
 
 
+def smape_func(y_true, y_pred) -> float:
+    epsilon = np.finfo(np.float64).eps
+    return np.average(np.abs(y_pred - y_true) / np.maximum(np.abs(y_true) + np.abs(y_pred), epsilon))
+
+
+symmetric_mean_absolute_percentage_error = make_scorer("symmetric_mean_absolute_percentage_error", smape_func, optimum=0.0, greater_is_better=False)
+symmetric_mean_absolute_percentage_error.add_alias("smape")
```

Comment:
If we don't want to add SMAPE as a default metric for `problem_type=REGRESSION`, I will move the metric definition to `autogluon.timeseries`.